### ISSUE TITLE
HTML produced by sparky is not valid

### ISSUE THREAD

**Issue by thibaultduponchelle** [melezhik/sparky#21]
The HTML produced by spark web is weird. No html tag, script and css before head, head almost empty etc...
Can we fix it ?

**Comment by melezhik** [comment]
Yes. I've never bothered about it correctness as long as it works in browser 😄 .
